build.gradle

```diff
@@ -60,9 +60,11 @@ allprojects {
   targetCompatibility = '11'
 
   repositories {
+    mavenLocal()
```

Comment:
Yeah it's normally committed like that. It just allows you to build local copies of SNAPSHOT dependencies and have them be picked up.  We generally want to avoid using snapshot dependencies in checked in code.  May have to live with it for discovery for a bit but might do a 0.3 release of it with the latest changes so we can depend on that now.